airflow/providers/dbt/cloud/example_dags/example_dbt_cloud.py

```diff
@@ -22,7 +22,7 @@
 try:
     from airflow.operators.empty import EmptyOperator as DummyOperator
 except ModuleNotFoundError:
-    from airflow.operators.dummy import DummyOperator
+    from airflow.operators.dummy import DummyOperator  # type: ignore
```

Comment:
mypy is very picky in this case
```
Run mypy.................................................................................Failed
- hook id: mypy
- exit code: 1

airflow/providers/microsoft/azure/example_dags/example_adf_run_pipeline.py:25: error:
Incompatible import of "DummyOperator" (imported name has type
"Type[DummyOperator]", local name has type "Type[EmptyOperator]")  [misc]
```

followed https://stackoverflow.com/questions/59037244/mypy-incompatible-import-error-for-conditional-imports which recommanded to add `type: ignore` in this case.